tfe/tool_helpers.go

```diff
@@ -0,0 +1,42 @@
+package tfe
+
+import (
+	"fmt"
+
+	tfe "github.com/hashicorp/go-tfe"
+)
+
+// fetchTerraformVersionID returns a Terraform Version ID for the given Terraform version number
+func fetchTerraformVersionID(version string, client *tfe.Client) (string, error) {
```

Comment:
it might be worth returning the entire tool version struct since you have it

Reply:
What would the struct be used for? When a resource is imported, the provider fetches the resource using the given ID.  The goal of a custom importer function is to retrieve/parse the ID the provider will use to fetch the resource (even though you're technically fetching twice)